pkg/agent/openflow/pipeline.go

```diff
@@ -521,7 +521,7 @@ func (c *client) connectionTrackFlows(category cookie.Category) []binding.Flow {
 				Done(),
 			connectionTrackCommitTable.BuildFlow(priorityLow).MatchProtocol(binding.ProtocolIP).
 				MatchCTStateTrk(true).
-				MatchCTMark(serviceCTMark).
+				MatchCTMark(serviceCTMark, nil).
```

Comment:
I feel that we actually don't need to use a pointer here since it looks like `nil` here is equal to `0xffffffff`. Explicitly set the mask would be a better and safer practice IMO.

Reply:
My point is if using a "uint64" value, it requires developer should know what mask is required every time when calling the API. But for OVS command line usages, we don't need to set mask in most cases. @suwang48404  @jianjuns @tnqn  would you like to share your thoughts?

Reply:
both sound good to me, I know normally not setting a mask mean full mask so I'm fine with keeping it as is, but better to add a comment to the function to declare the behavior.

Reply:
Sure, then I would add a comment to explain that OVS would match the value exactly if the mask is nil.